public/docs/_examples/pipes/dart/lib/exponential_strength_pipe.dart

```diff
@@ -12,9 +12,16 @@ import 'package:angular2/angular2.dart';
  *   formats to: 1024
  */
 @Pipe(name: 'exponentialStrength')
-@Injectable() // FIXME(chalin): unnecessary?
 class ExponentialStrengthPipe extends PipeTransform {
-  num transform(num value, String exponent) =>
-    math.pow(value,
-      num.parse(exponent, onError: (_) => 1));
+  transform(dynamic value, [List<dynamic> args]) {
+    var v = int.parse(value.toString(), onError: (source) => 0);
+    var p = args.isEmpty
+        ? 1
+        : int.parse(args.first.toString(), onError: (source) => 1);
+    return math.pow(v, p);
+  }
+  /* beta.15+
```

Comment:
Please remove, we can easily find this code back if needed.

Link to Dart beta.16 update: https://github.com/angular/angular.io/commit/0557c728d75e519bc6846b37492414dff540c6ee